vlasovsolver/cpu_trans_map_amr.cpp

```diff
@@ -1231,7 +1236,7 @@ bool trans_map_1d_amr(const dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>&
                // Loop over cells in pencil +- 1 padded cell
                for ( uint celli = 0; celli < targetLength; ++celli ) {
                   
-                  Realv vector[VECL];
+                  Realf vector[VECL];
```

Comment:
Is this meant to have changed?

Reply:
I noticed that I was using Realv datatypes in some places for vdf values, and I thought I'd fix that as well.

Reply:
ok